exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,11 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = 15 + 8;
```

Comment:
Well Done! For total, you can use the variable names instead of their value. for example : 
**let total : numberOfStudents+numberOfMentors;**
and as it mentioned in Code Style Guide document, "You should never use var. This is an outdated way of writing JavaScript code and it is always better to use let or const."[https://syllabus.codeyourfuture.io/guides/code-style-guide](url)
Good Luck

Reply:
I have changed the var to let. Thank you very much. I appreciate you let me know and remember to read the syllabus.